server/app/models/task.js

```diff
@@ -0,0 +1,78 @@
+const { Model, DataTypes } = require('sequelize');
+const Joi = require('joi');
+const { AssociationChangesSchema } = require('./association-changes');
+
+/**
+ * This class represent a Task.
+ */
+class Task extends Model {
+
+    /**
+     * Initialization function.
+     *
+     * @param sequelize
+     * @returns {Model}
+     */
+    static init(sequelize) {
+        // eslint-disable-next-line
+        this.prototype.toJSON = function () {
```

Comment:
Nice tip :) We schould use it for all models.

Reply:
We should add this as a function of the model instead of changing manually his prototype (if it works)

Reply:
I don't understand why ?
If we add a function as you describe, we must call it every time: `res.json(komission.toJson())`.
By changing his prototype, it is automatic: `res.json(kommission)`.

Reply:
@Embraser01 ?

Reply:
Yep it's just a syntax problem, we should add it to the Model like any other method, not by editing the prototype object

Reply:
Same comment :)

>I don't understand why ?
If we add a function as you describe, we must call it every time: `res.json(komission.toJson())`.
By changing his prototype, it is automatic: `res.json(kommission)`

Reply:
I don't want to have `this.protoype.something = function () {...}` instead of 
the same semantic things : 
``````

Reply:
Ok I implement that !